index.bs

```diff
@@ -2253,7 +2253,7 @@ Consensus</h4>
 Managing Dissent</h4>
 
 	In some cases, even after careful consideration of all points of view,
-	a group might find itself unable to reach consensus.
+	a group might find itself unable to reach a consensus.
```

Comment:
We almost if not absolutely never say `reach a consensus`; rather, we leave out the `a`.
```suggestion
	a group might find itself unable to reach consensus.
```